frb_example/with_flutter/lib/bridge_definitions.dart

```diff
@@ -0,0 +1,102 @@
+// AUTO GENERATED FILE, DO NOT EDIT.
```

Comment:
Is my understanding correct:

* bridge_definitions.dart: Declarations without implementations
* bridge_generated.dart: Implementations that applies both to io and web
* bridge_generated.io.dart, bridge_generated.web.dart: Implementaions specific to io or web

If so maybe we can add a few sentences into our doc to briefly tell users what happens :)